### DISCUSSION TITLE
Client Routing Middleware

### DISCUSSION THREAD

**Discussion by psycongaroo** [nuxt/nuxt#18136] [Questions]
Why removing client routing middleware from Nuxt 3?
It was so useful.

**Comment by psycongaroo** [comment]


**Reply by psycongaroo** [reply]
bump

**Reply by psycongaroo** [reply]
bump

**Reply by kissu** [reply]
Hey, no real purpose on bumping things on your own message.
You could meanwhile track this one: https://github.com/nuxt/framework/discussions/2082
